cpp/src/Ice/ConnectionI.h

```diff
@@ -307,13 +307,19 @@ namespace Ice
         IceInternal::SocketOperation read(IceInternal::Buffer&);
         IceInternal::SocketOperation write(IceInternal::Buffer&);
 
-        // A connection is at rest if it is active and has no outstanding invocations or dispatches.
+        // A connection is at rest if:
+        // - it is active and has no outstanding invocations or dispatches.
+        // - it is not reading a message which is not a connection validation message
```

Comment:
Fixed comment to:
```
// A connection is at rest if:
// - it is active and has no outstanding invocations or dispatches.
// - it is not reading or writing a message other than a connection validation message
// We schedule the inactivity timer task when it enters the "at rest" state, and we cancel this timer task when
// the connection is about to leave this state. The writing or reading of a connection validation message isn't
// considered as application "activity" since it's used internally to check the connection liveliness.
// Must be called with _mutex locked.
```